budget-vs-actual/provincial/pipeline-spec.yaml

```diff
@@ -31,211 +33,199 @@ budget-vs-actual-provincial:
+    - run: delete_fields
       parameters:
-        resources:
-          - epre-2019-20
-          - epre-2018-19
-          - epre-2017-18
-          - epre-2016-17
-          - epre-2015-16
+        resources: epre-2015-16
         fields:
-          -
-            operation: constant
-            target: AmountKind
-            with: 'Total'
+          - VoteNumber
+          - SubprogNumber
+          - Subprogramme
 
-    - run: add_computed_field
+    ### ARE 2016-17
+    - run: load
       parameters:
-        resources:
-          - epre-2017-18
-          - epre-2016-17
-          - epre-2015-16
-        fields:
-        - operation: format
-          target: corrected_econ2
-          with: '{economic_classification_1}'
+        from: 'https://s3-eu-west-1.amazonaws.com/manual-uploads.vulekamali.gov.za/are/are-2016-17-processed-2019-08-26t1500.csv'
+        name: 'are-2016-17'
+        format: 'csv'
+      cache: true
 
-    - run: add_computed_field
+    - run: set_types
       parameters:
-        resources:
-          - epre-2017-18
-          - epre-2016-17
-          - epre-2015-16
+        resources: are-2016-17
+        types:
+          Value:
+            type: number
+
+    - run: filter
+      parameters:
+        resources: are-2016-17
+        in:
+          - FinancialYear: '2016'
+
+    - run: delete_fields
+      parameters:
+        resources: are-2016-17
         fields:
-        - operation: format
-          target: corrected_econ3
-          with: '{economic_classification_2}'
+          - VoteNumber
+          - SubprogNumber
+          - Subprogramme
 
-    - run: add_computed_field
+    ### ARE 2015-16
+    - run: load
       parameters:
-        resources:
-          - epre-2017-18
-          - epre-2016-17
-          - epre-2015-16
+        from: 'https://s3-eu-west-1.amazonaws.com/manual-uploads.vulekamali.gov.za/are/are-2015-16-processed-2019-08-26t1500.csv'
+        name: 'are-2015-16'
+        format: 'csv'
+      cache: true
+
+    - run: set_types
+      parameters:
+        resources: are-2015-16
+        types:
+          Value:
+            type: number
+
+    - run: filter
+      parameters:
+        resources: are-2015-16
+        in:
+          - FinancialYear: '2015'
+
+    - run: delete_fields
+      parameters:
+        resources: are-2015-16
         fields:
-        - operation: format
-          target: corrected_econ4
-          with: '{economic_classification_3}'
+          - VoteNumber
+          - SubprogNumber
+          - Subprogramme
 
     - run: add_computed_field
       parameters:
         resources:
+          - epre-2019-20
+          - epre-2018-19
           - epre-2017-18
           - epre-2016-17
           - epre-2015-16
+          - are-2016-17
+          - are-2015-16
         fields:
-        - operation: format
-          target: corrected_econ5
-          with: '{economic_classification_4}'
+          -
+            operation: constant
+            target: AmountKind
```

Comment:
here I think you are overwriting AmountKind == Virements with Total which will introuce error - I think what you mean to do is add AmountKind == Total filter instead of rewriting the value

Reply:
I added a filter for this.

https://github.com/vulekamali/treasury-pipelines/pull/26/commits/fe60ad49fb0a5673e4ec9bf8578ba3dc53cf93fa